js/index.js

```diff
@@ -24,10 +24,35 @@ const display = document.querySelector('[data-js="display"]');
 
 function getGreeting() {
   // Code here
+  const currentHour = new Date().getHours();
+  // console.log({currentHour});
+  if (currentHour >= 6 && currentHour <= 12){
+    return `Good Morning, It is ${currentHour} O\'clock`
+  } else if (currentHour >= 13 && currentHour <= 18) {
+    return `Good Afternoon, It is ${currentHour} O\'clock`
+  }else if (currentHour >= 19 && currentHour <= 22) {
+    return `Good Evening, It is ${currentHour} O\'clock`
+  }else if (currentHour >= 23 && currentHour <= 5) {
+    return `Good Night, It is ${currentHour} O\'clock`
+  }
+  return currentHour
 }
 
 function getDayColor() {
   // Code here
+  let day = new Date().getDay();
+  // console.log({day})
+  switch(day) {
+    case 1 : return "darkgray";
+    break;
+    case 2 : return "lightblue";
+    break;
+    case 3 : return "hotpink";
+    break;
+    case 4 : return "darkgray";
+    break;
+  }
+  return "white"
 }
```

Comment:
i think this does not return as expected? it will return "darkgrey" for monday (day 1) - which is correct - but then it is supposed to return "lightblue" for days 2 to 5 (tuesday to friday) and then "hotpink" for days 6 an 7 (the weekend) and this one covers only 4 days / 4 cases .  so if you do want to stick to the switch then a possible solution could be:

 function getDayColor() {

 let day = new Date().getDay();

  switch(day) {
    case 1 : return "darkgray";
    break;
    case 2 : return "lightblue";
    break;
    case 3 : return "lightblue";
    break;
    case 4 : return "lightblue";
    break;
    case 5 : return "lightblue";
    break;
    case 6 : return "hotpink";
    break;
    case 7 : return "hotpink";
    break;
}
    return "white";
  }